### PR TITLE
Weekly cleanup

### PR DESCRIPTION
**Pull request by djmitche**
Untested, but the syntax is correct at least..

### CODE REVIEW

**Comment by seankelly** [comment]
LGTM.

File: files/weekly-summary.py [hunk 1]
```diff
@@ -29,6 +29,21 @@
 FROM = 'dustin@buildbot.net'
 RECIPIENTS = ['devel@buildbot.net', 'users@buildbot.net']
 
+WEEKLY_MEETING_TEXT = textwrap.dedent("""
+
+    <p>Buildbot has weekly meetings via irc, held at 16:30 UTC on Tuesdays.
```
Comment (sa2ajj) [inline files/weekly-summary.py]: do we not want one-sentence-per-line here?

Comment (sa2ajj) [inline files/weekly-summary.py]: not exactly at this line, but in the change :smile:

**Comment by djmitche** [comment]
howzat?

**Pull request merged**